Distraction Shield/optionsPage/htmlFunctionality.js

```diff
@@ -1,68 +1,62 @@
-    html_parent.append(html_child);
-};
-
-prependHtmlItemTo = function(html_child, html_parent) {
-    html_parent.prepend(html_child);
-};
-
 /* -------------------- Button Click functions ----------------------- */
 
-saveNewUrl = function () {
-    var newUrl = html_txtFld.val();
-    blockedSiteBuilder.createNewBlockedSite(newUrl, addBlockedSiteToAll);
-    html_txtFld.val('');
-};
-
-//Connect functions to HTML elements
-connectButton = function(html_button, method) {
+/**
+ * connect function to onclick property of html_item
+ */
+export function connectButton(html_button, method) {
     html_button.on('click', method);
-};
-
+}
 /* -------------------- Keypress events ----------------------- */
 
-setKeyPressFunctions = function() {
-    submitOnKeyPress(html_txtFld);
-    deleteOnKeyPress(blacklistTable);
-};
+export function setKeyPressFunctions(html_txtFld, blacklistTable, submitFunc, deleteFunc) {
+    submitOnKeyPress(html_txtFld, submitFunc);
+    deleteOnKeyPress(blacklistTable, deleteFunc);
+}
 
-submitOnKeyPress = function(html_elem) {
-    html_elem.keyup(function(event) {
-        if (event.keyCode == KEY_ENTER) {
-            saveNewUrl();
+function submitOnKeyPress(html_elem, submitFunc) {
+    html_elem.keyup((event) => {
+        if (event.keyCode === constants.KEY_ENTER) {
+            submitFunc();
         }
     });
-};
+}
 
-deleteOnKeyPress = function(blacklistTable) {
-    $('html').keyup(function(e) {
-        if (e.keyCode == KEY_DELETE) {
-            var html = blacklistTable.getSelected();
-            removeBlockedSiteFromAll(html);
+function deleteOnKeyPress(blacklistTable, deleteFunc) {
+    $('html').keyup((e) => {
+        if (e.keyCode === constants.KEY_DELETE) {
+            let html = blacklistTable.getSelected();
+            deleteFunc(html);
         }
     });
-};
-
+}
 /* -------------------- Logic for the mode selection -------------------- */
 
-initModeSelection = function(buttonGroup) {
-    $("input[name=" + buttonGroup + "]").change( function(){
-        settings_object.setMode($("input[name=" + buttonGroup + "]:checked").val());
+export function initModeSelection(buttonGroup, settings_object) {
+    $("input[name=" + buttonGroup + "]").change(function () {
+        let selectedMode = $("input[name=" + buttonGroup + "]:checked").val();
+        if (selectedMode === 'pro') {
+            settings_object.mode = constants.modes.pro;
+        } else {
+            settings_object.mode = constants.modes.lazy;
+        }
         synchronizer.syncSettings(settings_object);
     });
-};
-
+}
 /* -------------------- Interval slider -------------------- */
 
-initIntervalSlider = function() {
-    intervalSlider = new GreenToRedSlider('#interval-slider', function(value) {
-        settings_object.setInterceptionInterval(parseInt(value));
+export function initIntervalSlider(settings_object) {
+    return new GreenToRedSlider('#interval-slider', function (value) {
```

Comment:
#interval-slider can be extracted into a constant, and all other custom id's and classes that you use - the same. Much easier when you update the html.